grails-app/i18n/messages.properties

```diff
@@ -1,4 +1,4 @@
-default.doesnt.match.message=Property [{0}] of class [{1}] with value [{2}] does not match the required pattern [{3}]
+1Gdefault.doesnt.match.message=Property [{0}] of class [{1}] with value [{2}] does not match the required pattern [{3}]
```

Comment:
`1Gdefault ` looks like a typo - haven't checked subsequent commits so might already be fixed.

Reply:
vi user detected. Fixed.